content/docs/command-reference/run.md

```diff
@@ -107,9 +107,10 @@ Relevant notes:
   defined as outputs every time its executed by DVC.
 
 - In some situations we have executed a stage and later notice that some of the
-  output files or directories it creates, which are already in the workspace,
-  are missing from `dvc.yaml` (`outs` field). We can
-  [add missing outputs to an existing stage](/docs/user-guide/how-to/add-output-to-stage)
+  existing dependencies of the stage or output files/directories it creates,
```

Comment:
Same. Deps are also files/dirs. Like this is sounds as if only outs are files/dirs.

Reply:
Also done